app/controllers/users/registrations_controller.rb

```diff
@@ -0,0 +1,38 @@
+# frozen_string_literal: true
+
+class Users::RegistrationsController < Devise::RegistrationsController
+  include RackSessionsFix
+  respond_to :json
+
+  def respond_with(current_user, _opts = {})
+    if resource.persisted?
+      render json: {
+        status: {code: 200, message: 'Signed up successfully.'},
+        data: UserSerializer.new(current_user).serializable_hash[:data][:attributes]
+      }
+    else
+      render json: {
+        status: {message: "User couldn't be created successfully. #{current_user.errors.full_messages.to_sentence}"}
```

Comment:
Eu tive que mudar isso para poder conseguir criar um usuario, e entrar no sistema, nao sei se ficou algo para tras

```suggestion
before_action :configure_sign_up_params, only: [:create]

  def create
    @user = User.new(sign_up_params)

    if @user.save
      render json: {
        status: { code: 200, message: 'Signed up successfully.' },
        data: UserSerializer.new(@user).serializable_hash[:data][:attributes]
      }
    else
      render json: {
        status: { message: "User couldn't be created successfully. " + @user.errors.full_messages.to_sentence }
```

Reply:
Tem como me passar o que tem no configure sign up params depois?
seria tipo esse permit no pokemons_controller.rb?
```
 def pokemon_params
    params.require(:pokemon).permit(:nome, :tipo, :imagem, moves: [])
  end
```